tests/Controller/EnMarche/CitizenProjectControllerTest.php

```diff
@@ -343,6 +343,31 @@ public function testAuthenticatedAdherentCanFollowCitizenProject()
         $this->assertFalse($this->seeRegisterLink($crawler, 0));
     }
 
+    public function testFeaturedCitizenProject()
+    {
+        /** @var CitizenProject $citizenProject */
+        $citizenProject = $this->getCitizenProjectRepository()->findOneByUuid(LoadCitizenProjectData::CITIZEN_PROJECT_1_UUID);
+
+        $this->assertFalse($citizenProject->isFeatured());
```

Comment:
Assertions can be joined but should be surrounded by empty lines to distinguish them from the logic code.